deploy/sre-prometheus/100-multiple-efs-operator-PrometheusRule.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: aws-efs-alerts
+    role: alert-rules
+  name: aws-efs-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: aws-efs-alerts 
+    rules:
+    - alert: MultipleVersionsOfEFSOperatorInstalled 
```

Comment:
Better name it with multiple versions of EFS CSI driver installed. Since the real issue here is the csi driver but not the operator